### DISCUSSION TITLE
Option to download the new 1.5VAE and load them in AUTOMAT1C's UI?

### DISCUSSION THREAD

**Discussion by giorgostrc** [TheLastBen/fast-stable-diffusion#309] [Ideas]
To do that now we have to manually upload the VAE file to our drive and add --vae-path "path/to/VAE" every time we run the colab.

**Comment by TheLastBen** [comment]
Nope, the new vae was included in the colab a week ago, it is automatically embedded in the downloaded CKPT, make sure you're using the latest colab, and check the box "redownload the original model" in case you didn't update it

**Reply by giorgostrc** [reply]
What if I'm using the UI with models I've made using your Dreambooth training colab? Is the VAE embedded in those too?

**Reply by TheLastBen** [reply]
yes it is also included in the default model downloaded in dreambooth colab

**Reply by giorgostrc** [reply]
alright, thank you for taking the time to clarify!
